README.md

```diff
@@ -225,6 +225,34 @@ Any arguments after the application name will be passed directly to the applicat
 
 ```
 
+Sometime pipx can consume arguments provided for application
+```
+> pipx run pycowsay --py
+
+usage: pipx run [-h] [--no-cache] [--pypackages] [--spec SPEC] [--verbose] [--python PYTHON]
+                [--system-site-packages] [--index-url INDEX_URL] [--editable] [--pip-args PIP_ARGS]
+                app ...
+pipx run: error: ambiguous option: --py could match --pypackages, --python
+```
+
+To prevent this put double dash `--` before APP. It will make pipx to ignore arguments to the right
```

Comment:
```suggestion
To prevent this put double dash `--` before APP. It will make pipx to forward the arguments to the right verbatim to the application:
```